frontend/pages/queries/edit/EditQueryPage.tsx

```diff
@@ -38,7 +39,7 @@ interface IEditQueryPageProps {
   params: Params;
   location: {
     pathname: string;
-    query: { host_ids: string; team_id?: string };
```

Comment:
renamed url parameter to host_id since we only allow a single host id

Reply:
I think this would break existing bookmarks (probably OK, just wanted to be sure that we considered the impact)